test/browser/plugins/index.js

```diff
@@ -0,0 +1,26 @@
+/// <reference types="cypress" />
```

Comment:
Do you need/want this plugin? Or is it just a placeholder?

Reply:
That's not a plugin, that's a directive that informs typescript to include the `cypress` global types in this file (for intellisense & such). Cypress generates a sample file with that comment block on the top and I left most of it alone.

https://www.typescriptlang.org/docs/handbook/triple-slash-directives.html